copy/multiple.go

```diff
@@ -12,11 +12,35 @@ import (
 	internalManifest "github.com/containers/image/v5/internal/manifest"
 	"github.com/containers/image/v5/manifest"
 	"github.com/containers/image/v5/signature"
+	digest "github.com/opencontainers/go-digest"
 	imgspecv1 "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/exp/slices"
 )
 
+// prepareEditList prepares a list of instances which needs to edited without setting their content,
+// reason is to be able to test `what and how to add/edit` in the manifest list
+func prepareEditList(instanceDigests []digest.Digest, options *Options) ([]internalManifest.ListEdit, int) {
+	imagesToCopy := len(instanceDigests)
+	if options.ImageListSelection == CopySpecificImages {
+		imagesToCopy = len(options.Instances)
+	}
```

Comment:
(Assuming we build a list of copies to perform, `imagesToCopy` will be just a `len(theReturnedList)` and wil not need to be computed separately.)

Reply:
Done.